InternationalizationSample/InternationalizationSample/FSViewController.m

```diff
@@ -11,14 +11,6 @@
 
 @implementation FSViewController
 
-@synthesize button = button_;
-
-#pragma mark - View lifecycle
-
-- (void)viewDidLoad {
-  [super viewDidLoad];
-}
-
 # pragma mark - Actions
 
 - (void)triggerInvite:(id)sender {
```

Comment:
I wonder if we should add a "Reset state" button, as the other samples? So the clients would know the "proper way" to check eligibilities.